src/App.css

```diff
@@ -1,38 +1,3 @@
 .App {
   text-align: center;
 }
```

Comment:
can you please, move the text alignment to the header component so other components are not affected.